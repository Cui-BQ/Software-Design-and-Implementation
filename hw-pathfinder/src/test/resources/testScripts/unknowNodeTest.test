CreateGraph graph1

AddNode graph1 n1
AddNode graph1 n2

#FindPath n1 and n3
FindPath graph1 n1 n3

#FindPath n0 and n2
FindPath graph1 n0 n2

#FindPath n1 and n3
FindPath graph1 n0 n3
