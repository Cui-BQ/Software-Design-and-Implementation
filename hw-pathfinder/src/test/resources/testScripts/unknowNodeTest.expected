created graph graph1

added node n1 to graph1
added node n2 to graph1

#FindPath n1 and n3
unknown node n3

#FindPath n0 and n2
unknown node n0

#FindPath n1 and n3
unknown node n0
unknown node n3